server/apps/api_call.go

```diff
@@ -0,0 +1,16 @@
+package apps
+
+import (
+	"github.com/pkg/errors"
+)
+
+func (s *Service) Call(call Call) (*CallResponse, error) {
+	switch {
+	case call.Wish != nil && call.Modal == nil:
```

Comment:
If one of these and only one of these must be specified, can we reduce to one field that is a bool?